woocommerce/woocommerce-ios
# Order status Mark IV: manual status change

Closes #121 

This PR builds upon @mindgraffiti 's work in  #732, and won't be ready for a proper review until #732 is merged. But in the meantime, any feedback would be more than welcome! Thanks @mindgraffiti for your work on the BL and data transfer layers, integrating the UI was super easy! 

This is how it looks in this first iteration:

![screen recording 2019-03-06 at 11 05 54 am mov](https://user-images.githubusercontent.com/2722505/53873983-c07be200-4001-11e9-8d75-b9a8a2d88226.gif)

cc @kyleaparker 

## Testing
- Checkout the branch, run the tests.
- Take a good look at the code.
- For extra credit, make sure your store supports custom order statuses.
- Navigate to your order list, select one order, tap the pencil button
- Change the order status
- Check that the order actually changes status in the store backend, and the new status is reflected in the Order Details and Order List views
- Change the status of an order again. Tap the "Undo" button in the Notice. The change should get undone.
- Check that the relevant Tracks events are sent when tapping the pencil button and when tapping Apply in the Order Status List.

Update release notes:
